bin/index.js

```diff
@@ -15,8 +15,8 @@ function getProcessArgs () {
   }, {})
 }
 
-function getPackageName (path) {
-  return path.split('/').pop()
+function getPackageName (pat) {
```

Comment:
Nothing wrong here, except that I don't like the variable name `pat` 😛. Maybe simply `p` or perhaps `packagePath`?

Reply:
renamed to `packagePath`